pkg/util/tracing/span.go

```diff
@@ -201,6 +201,19 @@ func (sp *Span) SetBaggageItem(restrictedKey, value string) {
 	sp.i.SetBaggageItem(restrictedKey, value)
 }
 
+// TraceID retrieves a span's trace ID. Used for testing toggling a trace's
```

Comment:
The first sentence is sufficient, there's no reason we couldn't use it for something else.